### DISCUSSION TITLE
How to compile the docker-database container personally?

### DISCUSSION THREAD

**Discussion by Lostsite** [lobehub/lobehub#3748] [Development Question] [answered]
I was trying to use "lobehub/lobe-chat-database" docker, but I wish I can change some code to better fit my need (for example I switch "text-embedding-3-small" to "text-embedding-3-large" and some other minor changes). So I use git clone function to clone the repo and edited the code, than build a personal docker image, than deploy it.

Clearly the docker image I build does NOT support the database connection and user auth (even I have config the local postgresql page). 

Is there any instructions about how to personally compile the "lobehub/lobe-chat-database"?

**Comment by arvinxx** [comment, marked as answer]
```
docker build -t lobe-chat-database-local -f Dockerfile.database 
```
